import.go

```diff
@@ -355,3 +355,26 @@ func Import(c types.StackerConfig, storage types.Storage, name string, imports t
 
 	return nil
 }
+
+// Copy imports to a container rootfs
+func copyImportsInRootfs(name string, overlayDirs types.OverlayDirs, sc types.StackerConfig) error {
```

Comment:
I don't think this function name really makes sense, since it's not actually copying things into the rootfs. Maybe "copyOverlayDir" or similar?

Reply:
Ok, I will change